Library/Formula/ctags.rb

```diff
@@ -102,3 +103,22 @@ def caveats
  
  /*
  *   FUNCTION PROTOTYPES
+diff -ur a/ctags-5.8/routines.c b/ctags-5.8/routines.c
```

Comment:
Can you add some whitespace before beginning the new patch block and a comment with a link to this PR and the commit from which this code is derived? patch will ignore anything from the end of the last patch until the next `---`. You can get a link to a commit here: https://sourceforge.net/p/ctags/code/816/log/?path=/trunk